docker/compose/fca-low/core.yml

```diff
@@ -32,3 +34,29 @@ services:
       - data
     init: true
     command: 'pm2 logs'
+
+  init-core-fca-low:
```

Comment:
Not sure how to use in dev compared to prod, etc., would be happy to discuss about that with you and @fvo-dinum